kubejs/server_scripts/enigmatica/kubejs/base/recipes/replace_input.js

```diff
@@ -14,7 +14,7 @@ onEvent('recipes', (event) => {
     event.replaceInput({}, 'mythicbotany:elementium_ore', '#forge:ores/elementium');
     event.replaceInput({}, 'thermal:rubber', 'industrialforegoing:dryrubber');
     event.replaceInput({}, 'thermal:cinnabar', '#forge:gems/cinnabar');
-    event.replaceInput({}, 'thermal:sulfur', '#forge:gems/sulfur');
+    event.replaceInput({}, 'thermal:sulfur', '#forge:dust/sulfur');
```

Comment:
This might have some unintended consequences, like making sulfur blocks from dust instead of gems. And the sulfur + earth charge recipe becoming dust to dust.

Reply:
Mmmh, will check then

Reply:
Resolved in 41beae2